packages/cli/src/lib/builder/config.ts

```diff
@@ -132,6 +132,7 @@ export async function makeRollupConfigs(
               /\.woff$/,
               /\.woff2$/,
               /\.ttf$/,
+              /\.md/,
```

Comment:
Should end with md right?

```suggestion
              /\.md$/,
```